easybuild/easyconfigs/c/Cbc/Cbc-2.10.11-foss-2023a.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = "ConfigureMake"
+
+name = 'Cbc'
+version = '2.10.11'
+
+homepage = "https://github.com/coin-or/Cbc"
+description = """Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming
+solver written in C++. It can be used as a callable library or using a
+stand-alone executable."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/coin-or/Cbc/archive/refs/tags/releases']
+sources = ['%(version)s.tar.gz']
+checksums = ['1fb591dd88336fdaf096b8e42e46111e41671a5eb85d4ee36e45baff1678bd33']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('Doxygen', '1.9.7'),
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('METIS', '5.1.0'),
+    ('MUMPS', '5.6.1', '-metis'),
+    ('CoinUtils', '2.11.10'),
+    ('Osi', '0.108.9'),
+    ('Clp', '1.17.9'),
+    ('Cgl', '0.60.8'),
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.13'),
+]
+
+# Use BLAS/LAPACK from OpenBLAS
```

Comment:
Comment here is technically wrong since it's FlexiBLAS in this toolchain.
Better to just say use BLAS.. from toolchain.

Reply:
```suggestion
# Use BLAS/LAPACK from toolchain